packages/files/schema/models/FileRecord.js

```diff
@@ -0,0 +1,74 @@
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema, GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const FILE_RECORD_USER_META = '{ dv sender { dv fingerprint } authedItemId appId modelNames sourceAppId }'
+const FILE_RECORD_META_FIELDS = `{ id fileAdapter recordId path filename originalFilename mimetype encoding meta ${FILE_RECORD_USER_META} }`
+const FILE_RECORD_PUBLIC_META_FIELDS = `{ id recordId path filename originalFilename mimetype encoding meta ${FILE_RECORD_USER_META} }`
+
+
+const FileRecord = new GQLListSchema('FileRecord', {
+    schemaDoc: 'Stores uploaded file meta data and owner',
+    fields: {
+        fileMeta: {
```

Comment:
Why it's json? it's better to use plain fields filename, mimetype, originalFilename, path, encoding